lancelot.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Raizel",
+			"oem": "Redmi",
+			"device": "Redmi 9",
+			"filename": "crDroidAndroid-13.0-20230911-lancelot-v9.8.zip",
+			"download": "",
```

Comment:
use generated download url

Reply:
Generated download url is https://sourceforge.net/projects/rom-builders/files/lancelot/crDroidAndroid-13.0-20230911-lancelot-v9.8.zip/download

Should I use this ?

Reply:
Yes

Reply:
Done